master/docs/examples/hello.cfg

```diff
@@ -1,72 +1,71 @@
 # -*- python -*-
 # ex: set syntax=python:
 
-from buildbot.plugins import *
+from buildbot.plugins import buildslave
+from buildbot.plugins import changes
+from buildbot.plugins import util
+from buildbot.plugins import steps
```

Comment:
Preface: You're all welcome to modify and integrate these changes however you like.

As to why I write imports this way:
@marius-rizac Because Google's Style Guidelines require [each import have its own line](https://google.github.io/styleguide/pyguide.html#Imports_formatting)
@tardyp Because Google's Style Guidelines [implicitly prohibit them](https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports) (Only concrete imports are allowed) and [star imports do bad things](http://stackoverflow.com/q/2386714/1695680). Not to mention that you can still copy-paste this example and it still "just-works":tm:, I consider there to be _less_ to worry about importing when what to import is actually documented. :bow: